src/commands/synthetics/__tests__/utils.test.ts

```diff
@@ -46,6 +46,11 @@ describe('utils', () => {
   describe('runTest', () => {
     const processWrite = process.stdout.write.bind(process.stdout)
     const fakeTest = {
+      config: {
+        request: {
+          url: 'http://example.org/',
+        },
+      },
```

Comment:
This test was missing this making it [throw in `runTests`](https://github.com/DataDog/datadog-ci/blob/19836a630f094797880f6df367b119fc0125ec03/src/commands/synthetics/utils.ts#L73), this was the reason [this test was successful](https://github.com/DataDog/datadog-ci/pull/150/files#diff-f1ccd70098fb2de33a36d62bcfdefdedef3440975916ed0812c55b55fe9fe7ecL103) and not because of the expected `No test to trigger error`.